### DISCUSSION TITLE
It is possible to turn off the mouse?

### DISCUSSION THREAD

**Discussion by lzap** [nesbox/TIC-80#2102] [Q&A] [answered]
My game does not support it, the cursor if often disturbing. Thanks!

**Comment by nesbox** [comment, marked as answer]
Yes, try to call `poke(0x3FFB,0)`
https://github.com/nesbox/TIC-80/wiki/RAM#mouse-cursor

**Reply by atesin** [reply]
... yes....  on *every* TIC()
